debian-config-jobs

```diff
@@ -916,6 +916,8 @@ unset selection
 			sed -i "/^export ZSH=.*/a export ZSH_CACHE_DIR=~\/.oh-my-zsh\/cache" /etc/skel/.zshrc
 			# define theme
 			sed -i 's/^ZSH_THEME=.*/ZSH_THEME="risto"/' /etc/skel/.zshrc
+			# disable prompt while update
+			sed -i 's/# DISABLE_UPDATE_PROMPT="true"/DISABLE_UPDATE_PROMPT="true"/g' a.zsh
```

Comment:
Correct file name/location? Probably it should be /etc/skel/.zshrc ?

Reply:
corrected